README.md

```diff
@@ -7,37 +7,39 @@ These benchmarks are based on a similar project by
 
 _**Note**: at the moment the ExCALIBUR benchmarks are a work-in-progress._
 
+## Before you start
+
+If you are using spack for other projects, you might have to clear the cache if it exists:
+`rm -rf ~/.spack/cache`
+
 ## Requirements
 
 ### Spack
 
-_**Note**: we require Spack 0.18.  In some HPC facilities there may be already a
-central Spack installation available.  In principle you should be able to use a
-system installation of Spack (you only need to have `spack` in the `PATH`), but
-you need to make sure the version is recent enough to install the required
-packages and understand the provided environments.  Instructions below show you
-how to install Spack locally._
+_**Note**: In some HPC facilities there may be already a central Spack installation available.
+However, the version installed is most likely too old to support all the features
+used by this package. Therefore we recommend you install the latest version locally, 
+following the instructions below._
 
 [Spack](https://spack.io/) is a package manager specifically designed for HPC
-facilities.  Follow the [official
+facilities.  The [official
 instructions](https://spack.readthedocs.io/en/latest/getting_started.html) to
-install the latest version of Spack.
+install the latest version of Spack are summarised here for convenience.
+- git clone spack:
+`git clone -c feature.manyFiles=true https://github.com/spack/spack.git`
+- run spack setup script: `source ./spack/share/spack/setup-env.sh`
+- check spack is in `$PATH`, for example `spack --version`
```

Comment:
I'm not against putting the instructions here, but for context, the reason why I didn't copy them was not to duplicate information which is already in the official manual and which has a (admittedly very low) risk to become outdated.